File: tests/fixtures/code_samples/java/sample_029.java
// Sample 29: small utility.
package samples;

import java.util.List;

public final class Sample029 {
    private Sample029() {}

    public static int operation(List<Integer> xs) {
        int total = 29;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 29) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_029.py
"""Sample 29: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_29(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 29."""
    total = 29
    for x in xs:
        total += int(x)
    return total


def operation_29_pure(value: int) -> int:
    return (value * 29) % 7919

